.spice/datasets/prices_detailed.yaml

```diff
@@ -0,0 +1,13 @@
+name: spiceai.datasets.prices_detailed
```

Comment:
Do we want to expose raw views like this? Currently we expose a minute level roll up for each pair and one for all prices